cmd/utils.go

```diff
@@ -160,6 +161,24 @@ var globalProfiler interface {
 	Stop()
 }
 
+// Prints a given message after a requested delay.
+func delayPrint(message string, delay time.Duration) chan bool {
```

Comment:
I would prefer not printing it all the time. What we can do is quickly get @donatello 's vote and move forward.